Tests/Components Tests/Doku/DokuComponentTests.swift

```diff
@@ -178,12 +178,13 @@ class DokuComponentTests: XCTestCase {
     private var shopperInformation: PrefilledShopperInformation {
         let billingAddress = PostalAddressMocks.newYorkPostalAddress
         let deliveryAddress = PostalAddressMocks.losAngelesPostalAddress
-        let shopperInformation = PrefilledShopperInformation(shopperName: ShopperName(firstName: "Katrina", lastName: "Del Mar"),
-                                                             emailAddress: "katrina@mail.com",
-                                                             telephoneNumber: "1234567890",
-                                                             billingAddress: billingAddress,
-                                                             deliveryAddress: deliveryAddress,
-                                                             socialSecurityNumber: "78542134370")
-        return shopperInformation
+        return .init(
```

Comment:
@neelSharma12 I resolved a couple of warnings regarding `telephoneNumber` could you please give these a thorough look? :)

Reply:
We can use new field "phoneNumber" only for "PrefilledShopperInformation".